app/email_batching_test.go

```diff
@@ -341,7 +308,5 @@ func TestRenderBatchedPostFull(t *testing.T) {
 	}
 
 	var rendered = th.Server.renderBatchedPost(notification, channel, sender, "http://localhost:8065", "", translateFunc, "en", model.EMAIL_NOTIFICATION_CONTENTS_FULL)
-	if !strings.Contains(rendered, post.Message) {
-		t.Fatal("Rendered email should contain post contents when email notification contents type is set to Full.")
-	}
+	require.Contains(t, post.Message, rendered, "Rendered email should contain post contents when email notification contents type is set to Full.")
```

Comment:
I think you have to switch the order of the `post.Message` and `rendered` parameters.